### PR TITLE
feat(formgen): change output directory of model documents in formgen

### PR DESCRIPTION
**Pull request by sdstolworthy**
*Issue #, if available:*

*Description of changes:*

The option to customize the output directory of graphql documents used by forms is removed. Graphql documents consumed by generated forms are always generated in a nested graphql directory inside the outDir of the forms.

Additionally, --ui-out-dir has been renamed to --out-dir to maintain consistency with the other commands.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/form-generator/src/local_codegen_graphql_form_generator.ts [hunk 1]
```diff
@@ -98,20 +98,23 @@ export class LocalGraphqlFormGenerator implements GraphqlFormGenerator {
       renderTypeDeclarations: true,
       apiConfiguration: {
         dataApi: 'GraphQL',
-        fragmentsFilePath: path.join(
+        fragmentsFilePath: `./${path.join(
```
Comment (alharris-at) [inline packages/form-generator/src/local_codegen_graphql_form_generator.ts:101]: Does this work on windows?

Comment (sdstolworthy) [reply packages/form-generator/src/local_codegen_graphql_form_generator.ts:101]: This string is generated straight into the UI Form. If it is left out, the string is sent as `graphql/fragments`, which node attempts to resolve as a node module. I can get a windows vm set up to verify this.

Comment (sobolk) [reply packages/form-generator/src/local_codegen_graphql_form_generator.ts:101]: Should we get some integration(better)/e2e(worse) test for this where we attempt to use generated code?

Comment (sdstolworthy) [reply packages/form-generator/src/local_codegen_graphql_form_generator.ts:101]: This is definitely a missing test vector. I'll get a test written for this.

File: packages/form-generator/src/local_codegen_graphql_form_generator.test.ts [hunk 2]
```diff
@@ -251,4 +254,36 @@ void describe('LocalCodegenGraphqlFormGenerator', () => {
     const forms = await l.generateForms();
     assert('writeToDirectory' in forms);
   });
+  const graphqlDirectories = [
```
Comment (sdstolworthy) [inline packages/form-generator/src/local_codegen_graphql_form_generator.test.ts:257]: @alharris-at @sobolk I added these tests which exercise the path generation logic. This is integration-level; it is calling the real codegen package and making assertions against the output.

File: packages/form-generator/src/local_codegen_graphql_form_generator.ts (hunk 1 above)
Comment (sdstolworthy) [reply packages/form-generator/src/local_codegen_graphql_form_generator.ts]: https://github.com/aws-amplify/amplify-backend/pull/650/files#r1393530504

File: packages/form-generator/src/local_codegen_graphql_form_generator.test.ts (hunk 2 above)
Comment (sdstolworthy) [reply packages/form-generator/src/local_codegen_graphql_form_generator.test.ts:257]: image
These tests are passing on windows

**Review by sobolk — APPROVED** [review]

**Review by alharris-at — APPROVED** [review]

**Pull request merged**